api/src/main/scala/hmda/api/http/institutions/UploadPaths.scala

```diff
@@ -81,7 +84,9 @@ trait UploadPaths extends InstitutionProtocol with ApiErrorProtocol with Submiss
         val uploadedF = byteSource
           .via(splitLines)
           .map(_.utf8String)
-          .runForeach(line => processingActor ! AddLine(uploadTimestamp, line))
+          //.runForeach(line => processingActor ! AddLine(uploadTimestamp, line))
+          .mapAsync(parallelism = flowParallelism)(line => (processingActor ? AddLine(uploadTimestamp, line)).mapTo[Persisted])
```

Comment:
Would it be possible to use `mapAsyncUnordered` here, since we don't care about the order of the lars?